package.json

```diff
@@ -1,34 +1,45 @@
 {
   "name": "cartocolor",
   "version": "4.0.1",
-  "description": "A module of CartoColors for browserify",
+  "description": "CARTO custom color palettes for use with maps",
```

Comment:
The truth is that colors are indeed used now for some other things (eg. bold palette in design system for certain elements such as charts or even general UI things, like labels). I would probably suggest to keep original CartoColors, but dropping the browserify part... or a mix between both.... eg: "CartoColors: custom color palettes from CARTO"

Reply:
Sounds good, done!